addons/mail/static/src/xml/activity.xml

```diff
@@ -1,113 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
-    <t t-name="mail.activity_items">
-        <div class="o_thread_date_separator o_border_dashed" data-toggle="collapse" data-target="#o_chatter_planned_activities">
```

Comment:
There are probably some CSS rules that can be removed together with this template, but it should be checked on a case by case whether removed class names are used elsewhere.